notifications.bs

```diff
@@ -211,6 +217,11 @@ string <var>title</var>, {{NotificationOptions}} <a for=/>dictionary</a> <var>op
  <li><p>Set <var>notification</var>'s <a for=notification>body</a> to
  <var>options</var>["{{NotificationOptions/body}}"].
 
+ <li><p>If <var>options</var>["{{NotificationOptions/url}}"] <a for=map>exists</a>, then
+ <a lt="URL parser">parse</a> it using <var>baseURL</var>, and if that does not return failure, set
```

Comment:
Should we ban certain kinds of URLs, e.g. non-HTTP/S ones? What about cross origin URLs? (Technically the latter is allowed when using event listeners, though.)